homeassistant/components/kostal_plenticore/strings.json

```diff
@@ -4,7 +4,15 @@
       "user": {
         "data": {
           "host": "[%key:common::config_flow::data::host%]",
-          "password": "[%key:common::config_flow::data::password%]"
+          "password": "[%key:common::config_flow::data::password%]",
+          "service_code": "Service Code"
+        }
+      },
+      "reconfigure": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "password": "[%key:common::config_flow::data::password%]",
+          "service_code": "Service Code"
```

Comment:
```suggestion
          "service_code": "[%key:component::kostal_plenticore::config::step::user::data::service_data%]"
```

Reply:
it has to be service_code instead of service_data, right?

Reply:
Yes, good catch